library/src/com/squareup/timessquare/MonthView.java

```diff
@@ -26,6 +27,19 @@ public static MonthView create(ViewGroup parent, LayoutInflater inflater,
         dayBackgroundResId, dayTextColorResId, titleTextColor, displayHeader, headerTextColor,
         null);
   }
+    private static boolean isRTL() {
+        final int directionality = Character.getDirectionality(Locale.getDefault().getDisplayName().charAt(0));
```

Comment:
This may be problematic: http://stackoverflow.com/questions/18996183/identifyng-rtl-language-in-android#comment45214828_23203698

Change this line to:
```
String localeName = Locale.getDefault().getDisplayName();
if (localeName == null || localeName.isEmpty()) {
  return false;
}
final int directionality = Character.getDirectionality(localeName.charAt(0));
```